src/test/java/com/example/starter/its/IndexPageIT.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.example.starter.its;
+
+import com.flowlogix.testcontainers.PayaraServerLifecycleExtension;
+import com.flowlogix.util.ShrinkWrapManipulator;
+import org.eu.ingwar.tools.arquillian.extension.suite.annotations.ArquillianSuiteDeployment;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.drone.api.annotation.Drone;
+import org.jboss.arquillian.junit5.ArquillianExtension;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.jboss.arquillian.graphene.page.Location;
+import org.jboss.arquillian.graphene.page.Page;
+import org.jboss.arquillian.graphene.Graphene;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(PayaraServerLifecycleExtension.class)
+@ExtendWith(ArquillianExtension.class)
+@ArquillianSuiteDeployment
+class IndexPageIT {
+    @Drone
+    WebDriver browser;
+
+    @Page
+    IndexPage indexPage;
+
+    @Test
+    void h1HasText() {
+        Graphene.goTo(IndexPage.class);
+        assertThat(indexPage.h1.getText()).isEqualTo("Heading for Starter Project");
+    }
+
+    @Deployment
+    @SuppressWarnings("unused")
+    static WebArchive deploy() {
+        return ShrinkWrapManipulator.createDeployment(WebArchive.class);
+    }
+
+    @Location("index.xhtml")
+    static class IndexPage {
```

Comment:
While this is a good solution, there is a much easier way without creating an extra class. Can you figure that out?